README.md

```diff
@@ -10,3 +10,38 @@ Also add a picture of the structure to this [README](README.md) file.
 
 ### Well Entrance
 ![Well Entrance](https://github.com/Terasology/Mineshafts/raw/master/preview/WellEntrance.jpg)
+
+### Stage
+![Stage](https://github.com/tech45/AdditionalStructureTemplates/blob/master/preview/Stage.png)
+
+### Waterpool
+![Waterpool](https://github.com/tech45/AdditionalStructureTemplates/blob/master/preview/Waterpool.png)
+
+### Waterpool View - 2
+![Waterpool view-2](https://github.com/tech45/AdditionalStructureTemplates/blob/master/preview/Waterpool%20view-2.png)
+
+### More Waterpool using Structure Spawner
+![More Waterpool](https://github.com/tech45/AdditionalStructureTemplates/blob/master/preview/MoreWaterpools.png)
+
+
+# This is for my GCI Task (I) - Structure Templates.
+
+I have created Two new and awsome Structures!
```

Comment:
The readme is what people are going to see when they visit the repository. not sure why you need to refer to yourself in this way. can you remove this bottom section.

Reply:
Sir you can see that  I have done changes